model/src/main/java/org/jboss/pnc/model/BuildRecord.java

```diff
@@ -191,18 +191,6 @@ public class BuildRecord implements GenericEntity<Base32LongID> {
     @Column(updatable = false)
     protected Boolean scmBuildConfigRevisionInternal;
 
-    @Lob
-    @Type(type = "org.hibernate.type.TextType")
-    @Basic(fetch = FetchType.LAZY)
-    @LazyGroup("buildLog")
-    private String buildLog;
```

Comment:
You should add migration scripts to remove the fields from DB.

Reply:
Right! But let's keep it after the mgiration for a bit just to be safe.

Reply:
added model/src/main/resources/db-update/30000-remove-build-logs.sql